src/main/resources/scripts/kill counter.sk

```diff
@@ -2,25 +2,20 @@
 # Counts kills per life, in total, and the highest kill streak.
 #
 
-# Defaults are useful here, as a player's kills would be '<none>' if he hasn't killed anything yet,
-# but with defaults defined those will be used in that case.
-variables:
-	{kill counter.%player%.kills_session} = 0
-	{kill counter.%player%.kills_session_max} = 0
-	{kill counter.%player%.kills_total} = 0
-
-on death:
+death:
 	attacker is a player
-	add 1 to {kill counter.%attacker%.kills_total}
-	add 1 to {kill counter.%attacker%.kills_session}
-	{kill counter.%attacker%.kills_session} is greater than {kill counter.%attacker%.kills_session_max}:
-		set {kill counter.%attacker%.kills_session_max} to {kill counter.%attacker%.kills_session}
+	attacker isn't the victim
+	add 1 to {kill counter::%attacker's uuid%::kills}
+	add 1 to {kill counter::%attacker's uuid%::streak}
+	{kill counter::%attacker's uuid%::streak} > {kill counter::%attacker's uuid%::max}:
```

Comment:
Why remove advanced variables section usage? Is it not working?

Reply:
It's strange, but UUID's can't be used in the variables section.